src/components/atoms/OcButton/OcButton.vue

```diff
@@ -258,11 +258,11 @@ export default {
   align-items: center;
   border: 1px solid;
   border-color: transparent;
-  border-radius: 3px;
+  border-radius: 5px;
```

Comment:
Since you're changing it to our default border radius size, could you extend `oc-rounded` instead? Same for the other two lines where you changed the border radius to `5px`. I'd like to avoid writing the same hardcoded value throughout the code base and instead reuse / extend existing classes.

Reply:
yes good finding

Reply:
tried to use it wherever it was possible, for :hover i couldn't